decidim-core/lib/tasks/decidim_tasks.rake

```diff
@@ -36,4 +37,9 @@ namespace :decidim do
       decidim_verifications
     ).join(",")
   end
+
+  desc "Applies upgrade modifications to the already installed application."
+  task :upgrade_app do
```

Comment:
since this task is doing a very specific thing into removing the favicon, maybe it should be renamed to something specific like `remove_favicon`. 
I would expect a  decidim:upgrade_app to recreate the sidekiq.yml file, the initializer etc, so that new parameters to be added to the application configuration

Reply:
Applied at 03c22af99e118ad6ca4969d7a6803ec5c3b9ed93.